docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/FilterByKeyColumnFraction.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    using MulticlassClassificationExample = DatasetUtils.MulticlassClassificationExample;
+
+    public static class FilterByKeyColumnFraction
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<MulticlassClassificationExample> enumerableOfData = DatasetUtils.GenerateRandomMulticlassClassificationExamples(10);
+            var data = mlContext.Data.ReadFromEnumerable(enumerableOfData);
+
+            // Convert the string labels to keys
+            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("Label");
+            var transformedData = pipeline.Fit(data).Transform(data);
+
+            // Before we apply a filter, examine all the records in the dataset.
+            var enumerable = mlContext.CreateEnumerable<MulticlassWithKeyLabel>(transformedData, reuseRowObject: true);
+            Console.WriteLine($"Label\tFeatures");
```

Comment:
>Label [](start = 32, length = 5)

I kind of like this here is a sample of the dataset and here is the transformed data. Im wondering if we should update all samples to do something along these lines.